eslintrc.js

```diff
@@ -71,8 +93,12 @@ module.exports = {
       }
     ],
     "prefer-const": "error",
+    "prefer-object-spread": "error",
+    "prefer-regex-literals": "error",
+    "prefer-rest-params": "error",
+    "prefer-spread": "error",
+    "prefer-template": "error",
     "sort-requires/sort-requires": "error",
     "strict": ["error", "global"],
```

Comment:
BTW this seems sort of wrong. IMHO we should do this:

```
  "parserOptions": {
    "sourceType": "script"
  },
```

and set strict to safe which is the default.